-1

Tested at: 719b0d2cd458df76709e004dc41dc24c83beb842

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22248/26117/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>Selected class -> std::map<unsigned int, unsign>> Building LCG reflex dict from header file src/CondFormats/ESObjects/src/classes.h
>> Generating CondFormat Serialization code from header file src/CondFormats/ESObjects/src/headers.h 
Entering library rule at RecoLocalTracker/Records
Entering library rule at CalibTracker/Records
In file included from input_line_8:46:
/build/cmsbld/jenkins/workspace/ib-any-integration/CMSSW_10_1_X_2018-02-15-1100/src/DataFormats/ForwardDetId/interface/HGCEEDetId.h:37:46: error: cannot initialize return object of type 'ForwardSubdetector' with an rvalue of type 'DetId::Detector'
  ForwardSubdetector subdet() const { return HGCEE; }
                                             ^~~~~
Error: rootcling: compilation failure (tmp/slc6_amd64_gcc630/src/DataFormats/ForwardDetId/src/DataFormatsForwardDetId/a/DataFormatsForwardDetId_xrf35d8a354b_dictUmbrella.h)
gmake: *** [tmp/slc6_amd64_gcc630/src/DataFormats/ForwardDetId/src/DataFormatsForwardDetId/a/DataFormatsForwardDetId_xr.cc] Error 1
Entering library rule at DataFormats/SiStripCommon
</pre>

--- a/DataFormats/DetId/interface/DetId.h
+++ b/DataFormats/DetId/interface/DetId.h
@@ -22,7 +22,7 @@ public:
 
 
   enum Detector {Tracker=1, Muon=2, Ecal=3, Hcal=4, Calo=5, Forward=6,
-		 VeryForward=7, HGCEE=8, HGCHSi=9, HGCHSc=10 };
+		 VeryForward=7, HGCalEE=8, HGCalHSi=9, HGCalHSc=10 };
   /// Create an empty or null id (also for persistence)
   DetId()  : id_(0) { }
   /// Create an id from a raw number

